.github/workflows/CD.yml

```diff
@@ -94,6 +99,7 @@ jobs:
         script: |
           cd ${{ env.REPO_PATH }}
           echo "SITE_URL=${{ env.KAYUNGA_SITE_URL }}" > .env.nginx
+          echo "KAYUNGA_TAG_ID=${{ env.KAYUNGA_TAG_ID }}" >> .env.nginx
```

Comment:
You are adding the envs to the Nginx file here. See [link](https://github.com/devinit/ug-district-datahub/blob/dev/docker-compose.yml#L23) for location of env file in docker compose. 

I propose you add another secret for DB connection string (will share that with you). Add another line to write to the `.env` file not `.env.nginx`. Make sure to write both the DB connection string as well as the Analytics code.

Reply:
I don't have access to github settings on this repo, so I am unable to add secrets. Also why are we adding the DB connection string to github secrets?

Reply:
Adding them for security... We intend to make this repo public with time.